backend/tests/icpy/agent/tools/test_atlascloud_itv_tool.py

```
@@ -0,0 +1,571 @@
+    @pytest.mark.asyncio
+    async def test_save_video_local(self):
+        """Should save video to local workspace."""
+        tool = AtlasCloudImageToVideoTool()
+        video_bytes = b"test-video-data"
+        
+        mock_ws_service = AsyncMock()
+        mock_ws_service.get_workspace_path.return_value = "/tmp/workspace"
+        
+        with patch('icpy.agent.tools.context_helpers.get_workspace_service', return_value=mock_ws_service):
+            with patch('pathlib.Path.mkdir'):
+                with patch('builtins.open', mock_open()) as mock_file:
```

Ruff (0.14.14)

[error] Local variable `mock_file` is assigned to but never used

Remove assignment to unused variable `mock_file`

(F841)